src/jats/jats-exporter.ts

```diff
@@ -110,28 +96,36 @@ const findChildNodeOfType = (
   }
 }
 
-const isContributor = hasObjectType<Contributor>(ObjectTypes.Contributor)
-
-const CREDIT_VOCAB_IDENTIFIER =
-  'https://dictionary.casrai.org/Contributor_Roles'
-
-const chooseRoleVocabAttributes = (
-  role: ContributorRole
-): { [key: string]: string } => {
-  if (role.uri && role.uri.startsWith(CREDIT_VOCAB_IDENTIFIER)) {
-    return {
-      vocab: 'credit',
-      'vocab-identifier': CREDIT_VOCAB_IDENTIFIER,
-      'vocab-term': role.name,
-      'vocab-term-identifier': role.uri,
-    }
-  }
-
-  return {
-    vocab: 'uncontrolled',
-  }
+const findNodesOfType = (
```

Comment:
Can we use the prosemirror-utils package instead of implementing it ourselves? Also do we need to send to send the fragment? it looks like we always use `this.fragment`.

Reply:
Sure, i'll use the porsemirror-utils package.